util/src/util/Event.ts

```diff
@@ -5,6 +5,7 @@ import { EVENT, Event } from "../interfaces";
 const events = new EventEmitter();
 
 export async function emitEvent(payload: Omit<Event, "created_at">) {
+	console.log(payload) //TODO remove before merge
```

Comment:
remove it before merge xD

Reply:
Actually we should use the ``debug`` package to properly do logging. (we'll do this in separate PR)